src/components/Newsletter/NewsletterForm/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {FC} from 'react';
+import './index.scss';
+
+interface FormProps {
+  buttonType?: 'primary' | 'secondary';
+}
+
+const NewsletterForm: FC<FormProps> = ({buttonType = 'primary'}) => {
+  return (
+    <form
+      className="newsletter-form"
+      onSubmit={(e) => {
+        // TODO - Logic for saving the email for the upcoming newsletter
+        e.preventDefault();
+        console.log('Submitted');
+      }}
+    >
+      <input type="text" className="Input" placeholder="Email address" />
```

Comment:
Make it a controlled input and add some validations to it. Use yup for validations.